README.md

```diff
@@ -23,10 +23,10 @@ For this guide tomcat version 7 is used.
 `apt-get install openjdk-8-jdk`
 
 #### Clone Repository
-`git clone https://github.com/chreul/LAREX.git`
+`git clone git@github.com:OCR4all/LAREX.git`
```

Comment:
I'd just argue that the HTTPS path may be more user friendly.
Since people that want to use a SSH Key should know that they can replace this line, while people that do not know about SSH Keys may otherwise have difficulties cloning the repository.

I'll happily merge the request if you replace
git clone git@github.com:OCR4all/LAREX.git
with
git clone https://github.com/OCR4all/LAREX.git

Reply:
You are right. Sure thing, I have set it to clone with HTTPS. Thanks